src/lib/Microsoft.Health.Extensions.Fhir.R4/FhirClientFactory.cs

```diff
@@ -117,15 +129,16 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
 
                 if (Logger != null && !response.IsSuccessStatusCode)
                 {
+                    var errorType = ErrorType.FHIRServiceError;
                     var statusDescription = response.ReasonPhrase.Replace(" ", string.Empty);
 
                     if (response.StatusCode == System.Net.HttpStatusCode.TooManyRequests)
                     {
-                        Logger.LogMetric(FhirClientMetrics.HandledException($"FhirServerError{statusDescription}", ErrorSeverity.Informational, ConnectorOperation.FHIRConversion), 1);
+                        Logger.LogMetric(FhirClientMetrics.HandledException($"{errorType}{statusDescription}", ErrorSeverity.Informational), 1);

```

Comment:
nit: Sounds like only the severity is different in this if else clause so can simplify to :

var severity = response.StatusCode == System.Net.HttpStatusCode.TooManyRequests ? ErrorSeverity.Informational : ErrorSeverity.Critical;
Logger.LogMetric(...);

And probably then don't need that extra var for errorType.